pmm/pmm2-testsuite.groovy

```diff
@@ -128,26 +121,28 @@ pipeline {
         stage('Prepare') {
             steps {
                 deleteDir()
-                slackSend botUser: true, channel: '#pmm-ci', color: '#FFFF00', message: "[${JOB_NAME}]: build started - ${BUILD_URL}"
+                slackSend channel: '#pmm-ci', color: '#FFFF00', message: "[${JOB_NAME}]: build started - ${BUILD_URL}"
                 sh '''
-                    curl --silent --location https://rpm.nodesource.com/setup_7.x | sudo bash -
-                    sudo yum -y install nodejs
+                    curl -o - https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
+                    . ~/.nvm/nvm.sh
+                    nvm install 12.14.1
+                    sudo rm -f /usr/bin/node
+                    sudo ln -s ~/.nvm/versions/node/v12.14.1/bin/node /usr/bin/node
```

Comment:
actually, this doesn't work because sometimes the folder name contains '@'  (due to multiple instances running in parallel) and it errors out, so we can avoid it for now.